crates/bevy_core/src/time/fixed_timestep.rs

```diff
@@ -168,7 +168,7 @@ impl System for FixedTimestep {
         self.internal_system.is_send()
     }
 
-    unsafe fn run_unsafe(&mut self, _input: Self::In, world: &World) -> Self::Out {
+    unsafe fn run_unsafe(&mut self, (): (), world: &World) -> ShouldRun {
```

Comment:
nit: I consider the `_input` to `()` change to be a downgrade in legibility. `_input: ()` says "() is the input type, but it isn't used so we're ignoring it". `(): ()` says "this is nothing, look at the System trait to find out what that nothing is".

Reply:
Changed to `_input: ()`.